Classes/Emogrifier.php

```diff
@@ -195,7 +195,50 @@ public function emogrify()
             throw new \BadMethodCallException('Please set some HTML first before calling emogrify.', 1390393096);
         }
 
-        $xmlDocument = $this->createXmlDocument();
+        $xmlDocument = $this->process($this->createXmlDocument());
+        
+        return $xmlDocument->saveHTML();
+    }
+                
+    /**
+     * Applies the CSS you submit to the HTML you submit and returns only the
+     * HTML content within the <body> tag.
+     *
+     * This method places the CSS inline.
+     *
+     * @return string
+     *
+     * @throws \BadMethodCallException
+     */
+    public function emogrifyBodyContent()
+    {
+        if ($this->html === '') {
+            throw new \BadMethodCallException('Please set some HTML first before calling emogrify.', 1390393096);
+        }
+
+        $xmlDocument = $this->process($this->createXmlDocument());
+        
+        $innerDocument = new \DOMDocument();
+        foreach ($xmlDocument->documentElement->getElementsByTagName('body')->item(0)->childNodes as $childNode) {
+            $innerDocument->appendChild($innerDocument->importNode($childNode, true));
+        }
+        
+        return $innerDocument->saveHTML();
+    }
+    
+    /**
+     * Applies the submitted CSS to the submitted HTML.
+     *
+     * This method places the CSS inline.
+     *
+     * @param \DOMDocument $xmlDocument
+     *
+     * @return \DOMDocument
```

Comment:
We don't need to return $xmlDocument as it is passed by kind-of-reference and gets modified in the method. So the caller can just work on their reference after the call.